### ISSUE TITLE
Force Close

### ISSUE THREAD

**Issue by GoogleCodeExporter** [peteaky/android-xbmcremote#59]
```
What steps will reproduce the problem?

1. Enter Music
2. Use trackball to slide right in the menu.
3. When the menu marking goes out of screen I get Force Close.

What is the expected output? What do you see instead?

I expect that the menu would slide left.
Instead the menu remains still and the menu-marking goes out of screen and
I got Force Close.

What version of the product are you using? On what operating system?

App 0.5.11

Please provide any additional information below.

It works fine when tapping the screen.
Only when using the trackball.

```

Original issue reported on code.google.com by `bolin.andreas@gmail.com` on 3 Nov 2009 at 1:37

**Comment by GoogleCodeExporter** [comment]
```
Thanks!
```

Original comment by `phree...@gmail.com` on 3 Nov 2009 at 2:00
- Changed state: **Accepted**
- Added labels: **Priority-High**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Can you try with 0.5.13 and see if the issue is still there? Otherwise I'll 
close this.
```

Original comment by `phree...@gmail.com` on 4 Nov 2009 at 5:17

**Comment by GoogleCodeExporter** [comment]
```
This problem seems to be fixed on 0.5.13. Thanx!
```

Original comment by `bolin.andreas@gmail.com` on 4 Nov 2009 at 8:41

**Comment by GoogleCodeExporter** [comment]
```
Thanks for testing!
```

Original comment by `phree...@gmail.com` on 9 Nov 2009 at 9:47
- Changed state: **Fixed**
